yapf/yapflib/file_resources.py

```diff
@@ -52,11 +52,14 @@ def _GetExcludePatternsFromPyprojectToml(filename):
   """Get a list of file patterns to ignore from pyproject.toml."""
   ignore_patterns = []
   try:
-    import tomli as tomllib
```

Comment:
Not a fan of nested try/catches in favor of https://peps.python.org/pep-0484/#version-and-platform-checking style approach.

I'm also wondering if moving the import to ./yapf/yapflib/__init__.py is a good idea idea vs. conditional login in every file.

Lastly, this package is now part of the package requirements, so managing an `ImportError` may no longer be applicable.

```python
import sys
if sys.version_info >= (3, 10):
    import tomllib
else:
    import tomli as tomllib
```

if ImportError must still be handled.
```python
import sys
if sys.version_info >= (3, 10):
    import tomllib
else:
    try:
        import tomli as tomllib
    except ImportError:
        raise errors.YapfError(
            'tomli package is needed for using pyproject.toml as a '
            'configuration file')
```